packages/ubuntu_wsl_setup/pubspec.yaml

```diff
@@ -7,12 +7,17 @@ environment:
   flutter: ">=3.0.0"
 
 dependencies:
+  diacritic: ^0.1.3
   flutter:
     sdk: flutter
   flutter_html: ^3.0.0-0
+  form_field_validator: ^1.1.0
+  path: ^1.8.0
```

Comment:
remove the same line from `dev_dependencies` below:
```
warning • The dev dependency on path is unnecessary because there is also a normal dependency on that package • pubspec.yaml:46:3 • unnecessary_dev_dependency
```

Reply:
✅

Reply:
✅